lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -12,6 +11,9 @@
 logger = logging.getLogger("vetext_incoming_forwarder_lambda")
 logger.setLevel(logging.DEBUG)
 
+# http timeout for calling vetext endpoint
+HTTPTIMEOUT = 6
+
 
 def vetext_incoming_forwarder_lambda_handler(event: dict, context: any):
```

Comment:
What I'm getting from VA Profile for a request body, via the alb, is a JSON bytes array.  From the API Gateway2 docs, I expected `event["body"]` to be a dictionary.  I modified my lambda code to accept a dictionary, string, or bytes array as the body.  You might want to add `assert isinstance(event, dict)` if you don't anticipate anything else.  I recommend live testing with a request forwarded by the alb if you have only relied on unit tests so far.

Reply:
Yeah, we've done a bunch of tests from Twilio -> ALB -> Lambda -> Vetext and the functionality is all there and correct.